### PR TITLE
Applied enum value to ride type flags.

### PR DESCRIPTION
**Pull request by xiane**
Applied enum value of ride type flags in peep.

### CODE REVIEW

File: src/peep/peep.c [hunk 1]
```diff
@@ -556,13 +556,13 @@ static void sub_68F41A(rct_peep *peep, int index)
...
-						peep_head_for_nearest_ride_with_flags(peep, 0x00800000);
+						peep_head_for_nearest_ride_with_flags(peep, RIDE_TYPE_FLAG_SELLS_FOOD);
 						break;
 					case PEEP_THOUGHT_TYPE_THIRSTY:
-						peep_head_for_nearest_ride_with_flags(peep, 0x01000000);
+						peep_head_for_nearest_ride_with_flags(peep, RIDE_TYPE_FLAG_SELLS_DRINKS);
 						break;
 					case PEEP_THOUGHT_TYPE_BATHROOM:
-						peep_head_for_nearest_ride_with_flags(peep, 0x00200000);
+						peep_head_for_nearest_ride_with_flags(peep, RIDE_TYPE_FLAG_PEEP_SHOULD_GO_INSIDE_FACILITY);
```
Comment (duncanspumpkin) [inline src/peep/peep.c]: I think i made a mistake here it should have been `0x02000000` i.e. RIDE_TYPE_FLAG_IS_BATHROOM

File: src/peep/peep.c [hunk 2]
```diff
@@ -9058,7 +9058,7 @@ static void peep_head_for_nearest_ride_with_flags(rct_peep *peep, int rideTypeFl
 		}
 	}
 
-	if ((rideTypeFlags & 0x002000000) && peep_has_food(peep)) {
+	if ((rideTypeFlags & RIDE_TYPE_FLAG_IS_BATHROOM) && peep_has_food(peep)) {
```
Comment (duncanspumpkin) [inline src/peep/peep.c]: Can you also define line 9056 as RIDE_TYPE_FLAG_IS_BATHROOM | RIDE_TYPE_FLAG_SELLS_FOOD | RIDE_TYPE_FLAG_SELLS_DRINKS

**Comment by duncanspumpkin** [comment]
Looks good and its caught a small mistake I made when writing the code originally.

**Comment by xiane** [comment]
Thank you. I'm going to apply your advices.

**Pull request merged**